internal/auth/refresh.go

```diff
@@ -0,0 +1,128 @@
+package auth
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/airbytehq/abctl/internal/k8s"
+	"github.com/pterm/pterm"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EnsureValidAuth loads credentials and refreshes if expired
+func EnsureValidAuth(ctx context.Context, k8sClient k8s.Client, namespace string) (*Credentials, error) {
+	// Load credentials from secret
+	secret, err := k8sClient.SecretGet(ctx, namespace, "abctl-auth")
```

Comment:
Initially these commands were under abctl. I was asked to move them to airbox. Not sure what to do with these kind of names, if the intention is that the commands could be moved back under abctl? Do I change them, or leave them?